docs/html/reference/pip_install.rst

```diff
@@ -540,7 +540,7 @@ Finding Packages
 
 pip searches for packages on `PyPI`_ using the
 `HTTP simple interface <https://pypi.org/simple/>`_,
-which is documented `here <https://setuptools.readthedocs.io/en/latest/easy_install.html#package-index-api>`_
+which is documented `here <https://setuptools.readthedocs.io/en/latest/deprecated/easy_install.html#package-index-api>`_
```

Comment:
We should probably remove direct references to `easy_install` and PEP 503, and link to [Simple repository API on Python Packaging User Guide](https://packaging.python.org/specifications/simple-repository-api/) instead.

Reply:
Makes sense. I adjusted the link with the most recent commit.